views/VideoView.js

```diff
@@ -0,0 +1,14 @@
+var VideoView = Backbone.View.extend({
+  className: 'main-video',
+
+  template: Handlebars.compile($('#video-template').html()),
+
+  initialize: function () {
+    this.listenTo(this.model, 'change:searchTerm', this.render);
```

Comment:
Why do the video models need to re-render when the searchTerm changes? Won't the view be destroy when the term is changed?